src/common/models/Projection.ts

```diff
@@ -62,12 +65,16 @@ export type DatasetId =
   | 'caseDensityRange'
   | 'smoothedDailyCases'
   | 'smoothedDailyDeaths'
+  | 'weeklyNewCasesPer100k'
   | 'rawDailyCases'
   | 'rawDailyDeaths'
   | 'rawHospitalizations'
   | 'smoothedHospitalizations'
   | 'rawICUHospitalizations'
-  | 'smoothedICUHospitalizations';
+  | 'smoothedICUHospitalizations'
+  | 'weeklyCovidAdmissionsPer100k'
+  | 'bedsWithCovidPatientsRatio'
+  | 'canCommunityLevel';
```

Comment:
We probably don't need `canCommunityLevel` here.  We only need it if we're going to use it with the `getDataset(dataset: DatasetId)` method which is only used for timeseries data.

Just as an FYI- The way `getDataset()` takes a string and then directly reads that property from the class is pretty hacky / fragile and if we were writing this from scratch I wouldn't do it like this.